src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/OverseasEntitySubmissionDto.java

```diff
@@ -73,6 +73,9 @@ public class OverseasEntitySubmissionDto {
     @JsonProperty(UPDATE_FIELD)
     private UpdateDto update;
 
+    @JsonProperty(IS_REMOVE_FIELD)
+    private Boolean isRemove;
```

Comment:
Wondering if we need a Remove block similar to the Update block. We'll need to store the 'remove' statement later on the journey. Would it be better to keep them together?

Reply:
Yes, we'll need a remove block later on but was thinking that the fundamental decision/info about whether this was a 'remove' submission sat at a higher-level, so outside of that block. If its value was `true`, then you would expect the remove block to be populated.

Reply:
Yes that makes sense.